TP/steps/18-form-b-solution/src/app/shared/badge/sfeir-badge.directive.ts

```diff
@@ -9,14 +9,18 @@ export class SfeirBadgeDirective implements OnInit {
   /**
    * Component constructor
    */
-  constructor(private _el: ElementRef, private _rd: Renderer2) {}
+  constructor(private elementRef: ElementRef, private renderer: Renderer2) {}
```

Comment:
suggestion(non-blocking): same as before, this vcan be a best practice to make them readlonly

Reply:
Done d1837cb33d5424ba6c881cab8cf0b259d3652040

Reply:
same here